Rock.Rest/Controllers/FinancialTransactionsController.Partial.cs

```diff
@@ -317,7 +317,33 @@ public DataSet GetContributionTransactions( int groupId, int? personId, [FromBod
         }
 
         /// <summary>
-        /// 
+        /// Gets the contribution transactions.
+        /// </summary>
+        /// <param name="groupId">The group unique identifier.</param>
+        /// <returns></returns>
+        /// <exception cref="System.Web.Http.HttpResponseException"></exception>
+        [Authenticate, Secured]
+        [HttpGet]
+        [EnableQuery]
+        [System.Web.Http.Route( "api/FinancialTransactions/GetTransactionsForGivingGroup/{personId}" )]
+        public IQueryable<FinancialTransaction> GetTransactionsForGivingGroup( int personId )
+        {
```

Comment:
@jbaxleyiii it would be more performant since I wouldn't have to query for the person first thing.  @azturner I will make the change - just need to know if we mean `int GivingGroupId` or `string GivingId`.  Plus I would like to know if those are the same concept.